src/nodes/Emoji.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * @copyright Copyright (c) 2021 Jonas Meurer <jonas@freesources.org>
+ *
+ * @author Jonas Meurer <jonas@freesources.org>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import { Node } from 'tiptap'
+import { insertText } from 'tiptap-commands'
+
+export default class Emoji extends Node {
```

Comment:
I think this should also work with using an extension which would be a more cleaner way from my perspective to handle only the command, see https://github.com/ueberdosis/tiptap/blob/v1/packages/tiptap-extensions/src/extensions/History.js#L41-L48 for example. Unless there is a reason for keeping a dedicated node type of course that I might not see ;)

Reply:
You're right, an extension is enough here - no need to make it a node. Changed now :blush: